javascript/updates_for_element.js

```diff
@@ -79,36 +45,39 @@ class UpdatesForElement extends HTMLElement {
     if (this.channel) this.channel.unsubscribe()
   }
 
-  async fetchAndUpdateForIdentifier (identifier) {
+  update () {
+    const identifier = this.getAttribute('identifier')
     const query = `updates-for[identifier="${identifier}"]`
     const blocks = document.querySelectorAll(query)
     if (blocks[0] !== this) return
 
     const template = document.createElement('template')
-
-    const response = await fetch(
-      this.hasAttribute('url') ? this.getAttribute('url') : window.location.href
-    )
-    const html = await response.text()
-
-    template.innerHTML = String(html).trim()
-    const fragments = template.content.querySelectorAll(query)
-    for (let i = 0; i < blocks.length; i++) {
-      activeElement.set(document.activeElement)
-      const operation = {
-        element: blocks[i],
-        html: fragments[i],
-        permanentAttributeName: 'data-ignore-updates',
-        focusSelector: null
-      }
-      dispatch(blocks[i], 'cable-ready:before-update', operation)
-      morphdom(blocks[i], fragments[i], {
-        childrenOnly: true,
-        onBeforeElUpdated: shouldMorph(operation)
+    fetch(this.url)
```

Comment:
any reason you don't like `async/await` ? 😉

Reply:
Not consciously avoiding it! Did I break something or miss something?

Reply:
My goals were to move away from using an event and to simplify debounce

Reply:
well I did refactor it to async/await because I think it's easier on the eye:

https://github.com/julianrubisch/cable_ready/pull/3/files#diff-5759727f85bbac57ca267669f5241238081d4fb0c3d39d98750001f8906ef91eL89

Reply:
Did I do that right?

Like I said, I didn't make the change consciously! I've never been super savvy with the async/await syntax.

This definitely looks better - anything that removes an indent level is usually a win.

Reply:
Lol yeah, lgtm